Source/WebCore/dom/TrustedType.cpp

```diff
@@ -302,12 +303,36 @@ ExceptionOr<String> requireTrustedTypesForPreNavigationCheckPasses(ScriptExecuti
         : nullString());
 }
 
-ExceptionOr<bool> canCompile(ScriptExecutionContext& scriptExecutionContext, JSC::CompilationType compilationType, String codeString, JSC::JSValue bodyArgument)
+ExceptionOr<bool> canCompile(ScriptExecutionContext& scriptExecutionContext, JSC::CompilationType compilationType, String codeString, const JSC::ArgList& args)
```

Comment:
FYI, more tests for that have landed at https://github.com/web-platform-tests/wpt/commit/1076fe42f0ea43b6b1ea3b7bb13307bbee5cfafb